pom.xml

```diff
@@ -26,6 +26,7 @@
   </scm>
 
   <properties>
+	<jdk.version>1.7</jdk.version>
```

Comment:
formatting ?

Reply:
If you're saying that I need to replace the tab with spaces, you're right!

If you're asking if this is a formatting change, it isn't. `hotels-oss-parent` provides jdk 8 by default now :)

Reply:
Yeah. I meant the first one.